### PR TITLE
Fixes issue #1610 by adding a method to send parameters for the paginator path pattern method 

### PR DESCRIPTION
**Pull request by kurenn**

### CODE REVIEW

**Comment by kurenn** [comment]
Is this ever going to be merged?

**Comment by bimusiek** [comment]
+1
I am adding parameters right now before creating RKPaginator. This could be very useful.

File: Code/Network/RKObjectManager.h [hunk 1]
```diff
@@ -860,6 +874,7 @@ RKMappingResult, RKRequestDescriptor, RKResponseDescriptor;
  */
 - (RKPaginator *)paginatorWithPathPattern:(NSString *)pathPattern;
 
+
```
Comment (segiddins) [inline Code/Network/RKObjectManager.h]: nuke this empty line

File: Tests/Logic/ObjectMapping/RKObjectManagerTest.m [hunk 2]
```diff
@@ -1670,4 +1670,32 @@ - (void)testThatNonmatchingClassesReturnNil
     expect(requestDescriptor).to.beNil();
 }
 
+- (void)testThatPaginatorPathPatternWithOutParamsDictionaryEvaluatesAgainstPaginator
+{
+    
+    NSString *requestString = @"users/filtered?page=:currentPage&per_page=:perPage";
+    RKPaginator *paginator = [[RKPaginator alloc] init];
```
Comment (segiddins) [inline Tests/Logic/ObjectMapping/RKObjectManagerTest.m]: this `RKPaginator` is unused

**Comment by segiddins** [comment]
Please rebase on development so travis can run properly

**Comment by segiddins** [comment]
This was done in https://github.com/RestKit/RestKit/pull/2174.

**Pull request closed without merging**